buildSrc/src/main/kotlin/Java9Modularity.kt

```diff
@@ -108,12 +112,14 @@ object Java9Modularity {
         compileTask: KotlinCompile,
         sourceFile: File
     ): TaskProvider<out KotlinJvmCompile> {
-        apply<KotlinBaseApiPlugin>()
+        apply<KotlinApiPlugin>()
         val verifyModuleTaskName = "verify${compileTask.name.removePrefix("compile").capitalize()}Module"
         // work-around for https://youtrack.jetbrains.com/issue/KT-60542
```

Comment:
Is comment about KT-60542 still relevant?

Reply:
For current Kotlin version in `dev` branch - 1.9.21 - yes.